nkpremices/Banka
# #164911319 An ​ admin​ / ​ staff​ can view list of all bank accounts.

#### What does this PR do?
Have the admin and staff dashboards

#### Description of Task to be completed?
* Create all necessary files for the admin/staff dashboards
* Design the admin/staff dashboard
* Add responsiveness on them
* Add a list of all user accounts on them 

#### How should this be manually tested?
After cloning the repo, log in as an admin/staff and you will see the admin/staff dashboards

#### Any background context you want to provide?
There are js files listening to clicks

#### What are the relevant pivotal tracker stories?
#164912163
#164912266
